bc/assets/templates/base.html

```diff
@@ -19,6 +19,9 @@
     <link rel="icon" href="{% static "svg/icon.svg" %}" type="image/svg+xml">
     <link rel="apple-touch-icon" href="{% static "png/apple-touch-icon.png" %}">
 
+    {% block head_script %}
+    {% endblock %}
+
```

Comment:
I thought these days it was preferred to put JS scripts after the HTML so they load after the HTML has been downloaded and displayed? This could be obsolete info when using a CDN over HTTP/2. Do you know?

Reply:
You're right about the script tag, I think most of the JS we use these days can load after the web page is rendered but there are still scripts that advise placing the tag within the `<head> … </head>` because they're critical to the page contents and should be loaded first such as analytics or authentication-type of scripts, that is not our case so I'll move this block into the  `<body> … </body>`.